sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -14,11 +16,22 @@
 from sklearn.svm import SVC
 from sklearn.multiclass import OneVsRestClassifier
 
+
 # Load the iris dataset and randomly permute it
 iris = datasets.load_iris()
 X, y = iris.data[:, 1:3], iris.target
 
 
+def test_unfitted():
+    """Raise error if VotingClassifier was not fitted"""
+    eclf = VotingClassifier(estimators=[('lr', LogisticRegression())])
+    assert_raises(NotFittedError, eclf.predict, X)
+    assert_raises(NotFittedError, eclf.transform, X)
+    try:
```

Comment:
@MechCoder Somehow I couldn't get it to work with the `predict_proba` call. I guess that's because `predict_proba` is property. Any tips on how to get that to work with `assert_raises`?